kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_scopes/Clock.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.spring.basic.spring_scopes;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+@Component
+@Scope("prototype")
+public class Clock {
+
+    private LocalTime time;
+
+    public Clock() {
+        this.time = LocalTime.now();
+        getTime();
+    }
+
```

Comment:
Czemu to nie moze zwracac LocalTime?

Reply:
zwraca LocalTime bez metody też.  Zamierzenie moje było by czas wyświetlał się w konsoli, później zrobiłem z tego metodę getTime. Finalnie ją zostawiłem i zrobiłem do niej test, ale bez niej też są spełnione warunki zadania.